### PR TITLE
Moved creation of topo.Server outside of tx_throttler.

### PR DESCRIPTION
**Pull request by erzel**
@alainjobart  PTAL.

### CODE REVIEW

**Comment by alainjobart** [comment]
You didn't have to change vttablet.go / vtcombo.go?

**Comment by erzel** [comment]
Oops. You're right. I've updated these files. I've code searched for all other callers of NewServer / NewTabletServer and couldn't find any others. PTAL.

File: go/cmd/vttablet/vttablet.go [hunk 1]
```diff
@@ -74,7 +74,8 @@ func main() {
 	}
 
 	// creates and registers the query service
-	qsc := tabletserver.NewServer()
+	ts := topo.Open()
```
Comment (sougou) [inline go/cmd/vttablet/vttablet.go]: Is this function idempotent, or can it be made so? If so, we can have TabletServer start this as a sub-service internally, which will be in line with the rest of the sub-services. We'll also reduce one dependency.

Comment (erzel) [inline go/cmd/vttablet/vttablet.go]: I don't think the function is idempotent, currently. It looks to me like it opens an RPC connection.

Comment (alainjobart) [inline go/cmd/vttablet/vttablet.go]: This function is creating a new object, it should actually be called 'New'. We used to have a single 'global' topo server, and that was a bad pattern that made a whole bunch of tests really hard to manage.

**Comment by alainjobart** [comment]
Change looks good, but can you make sure the unit tests pass locally before review? thanks!

**Comment by erzel** [comment]
Sorry Alain. I'll make sure the tests are passing next time. PTAL.

**Pull request merged**